test/unit/helpers/MeasureBundleHelpers.test.ts

```diff
@@ -791,49 +791,46 @@ describe('MeasureBundleHelpers tests', () => {
 
   describe('extractLibrariesFromLibraryBundle', () => {
     it('properly gets libraries from EXM130 library bundle using resourceID for rootLibRef', () => {
-      const measureBundle = getJSONFixture('EXM130-7.3.000-bundle-nocodes.json') as fhir4.Bundle;
+      const measureBundle = getJSONFixture('measure/measure-with-library-dependencies.json') as fhir4.Bundle;
+
       const libraryBundle: fhir4.Bundle = {
         resourceType: 'Bundle',
         id: 'EXM130-7.3.000-bundle',
         type: 'transaction'
       };
       libraryBundle.entry = measureBundle.entry?.filter(e => e.resource?.resourceType === 'Library');
 
-      const rootLibRef = 'Library/library-EXM130-7.3.000';
+      const rootLibRef = 'Library/library-TestRootLib';
       const { cqls, rootLibIdentifier, elmJSONs } = extractLibrariesFromLibraryBundle(libraryBundle, rootLibRef);
 
       expect(rootLibIdentifier).toStrictEqual({
-        id: 'EXM130',
-        system: 'http://fhir.org/guides/dbcg/connectathon',
-        version: '7.3.000'
+        id: 'TestRootLib',
+        version: '0.0.1'
       });
-      // The EXM130 test bundle has 7 libraries, including the root one
-      // BUT one of them is the FHIR model info file, which we ignore
-      expect(cqls).toHaveLength(6);
-      expect(elmJSONs).toHaveLength(6);
+      // The test bundle has 3 libraries, including the root one
+      expect(cqls).toHaveLength(3);
+      expect(elmJSONs).toHaveLength(3);
     });
 
     it('properly gets libraries from EXM130 library bundle using canonical URL for rootLibRef', () => {
-      const measureBundle = getJSONFixture('EXM130-7.3.000-bundle-nocodes.json') as fhir4.Bundle;
+      const measureBundle = getJSONFixture('measure/measure-with-library-dependencies.json') as fhir4.Bundle;
       const libraryBundle: fhir4.Bundle = {
         resourceType: 'Bundle',
         id: 'EXM130-7.3.000-bundle',
         type: 'transaction'
       };
       libraryBundle.entry = measureBundle.entry?.filter(e => e.resource?.resourceType === 'Library');
 
-      const rootLibRef = 'http://fhir.org/guides/dbcg/connectathon/Library/EXM130';
+      const rootLibRef = 'http://example.com/Library/library-TestRootLib';
       const { cqls, rootLibIdentifier, elmJSONs } = extractLibrariesFromLibraryBundle(libraryBundle, rootLibRef);
 
       expect(rootLibIdentifier).toStrictEqual({
-        id: 'EXM130',
-        system: 'http://fhir.org/guides/dbcg/connectathon',
-        version: '7.3.000'
+        id: 'TestRootLib',
+        version: '0.0.1'
       });
-      // The EXM130 test bundle has 7 libraries, including the root one
-      // BUT one of them is the FHIR model info file, which we ignore
-      expect(cqls).toHaveLength(6);
-      expect(elmJSONs).toHaveLength(6);
+      // The EXM130 test bundle has 3 libraries, including the root one
```

Comment:
```suggestion
      // The test bundle has 3 libraries, including the root one
```
Something like that?